### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by stephanebenayoun**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Describe the bug
+
+A clear and concise description of what the bug is.
+
+## Reproduce
+
+#### Expected behavior
+
+A test case in _**Gherkin Syntax**_ or the link to this test case.
```
Comment (stephanebenayoun) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Add link and example

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Additional information
+
+#### Frequency
+
+100% reproduce / happens occasionally / hard to reproduce
+
+#### Attachments
+
+Screenshots, videos, documents, localStorage and cookies data etc..
```
Comment (stephanebenayoun) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Screenshots, videos, documents, log etc.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+#### Environment
+
+- OS & Version: [e.g. iOS:11.10...]
+- Browser & Version [e.g. chrome:x..., safari:y..] if relevant
```
Comment (stephanebenayoun) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- Node version
- Browser & Version [e.g. chrome:x..., safari:y..] if relevant
```

File: .github/ISSUE_TEMPLATE/change_request.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Provide link to a PR or code snippet of old and new API.
+
+## Behavior
+
+#### Consideration
+List of everything to consider before writing test cases.
+
+- [ ] Does the suggested changes impact the current behaviour ?
+<!-- If yes, specify which behaviour will be changed and how or provide the link to a PR.-->
```
Comment (dmitriyslaym) [inline .github/ISSUE_TEMPLATE/change_request.md]: I think you will not actually see these comments in a template itself

File: .github/ISSUE_TEMPLATE/change_request.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+## Ready for implementation
+- [ ] External dependencies have been resolved
+- [ ] API has been approved
+- [ ] Test cases have been prepared
+- [ ] Discussed with Product owner
+- [ ] Discussed with Technical lead
+
+## Definition of Done
+- [ ] Maintainer review
```
Comment (dmitriyslaym) [inline .github/ISSUE_TEMPLATE/change_request.md]: Who is maintainer in this case?

Comment (MikeSquall) [reply .github/ISSUE_TEMPLATE/change_request.md]: People that have rights on the repo and are able to set some labels

File: .github/ISSUE_TEMPLATE/change_request.md (hunk 4 above)
Comment (MikeSquall) [reply .github/ISSUE_TEMPLATE/change_request.md]: You will see it on the template, not in the preview / post

**Pull request merged**